redis/redis.go

```diff
@@ -63,6 +63,14 @@ func ClientFromConnectionString(
 	if err != nil {
 		return nil, err
 	}
```

Comment:
are we sure we will not catch this error condition first in case there is no "redis://" prefix? (asking since I do not know)

Reply:
parsing host:port will not cause an error, but it will be parsed as scheme:opaque[?query][#fragment], so at the end scheme will be equal host and host will be empty